torch/_inductor/triton_heuristics.py

```diff
@@ -820,7 +843,11 @@ def pointwise(size_hints, meta, tile_hint=None, filename=None):
         ):
```

Comment:
for fewer changes you could consider `triton_config = functools.partial(triton_config, min_elements_per_thread=min_elements_per_thread`